File: CPP-Language/FlowControl/if...else-statements/IfStatement.cpp
// Program to print positive number entered by the user
// If the user enters a negative number, it is skipped

#include <iostream>

using namespace std;

int main() {
    int number;

    cout << "Enter an integer number: ";
    cin >> number;

    //checks if the number is positive
    if (number > 0) {
        cout << "You entered a positive integer: " << number << endl;
    }
    cout << "This statement is always executed." << endl;

    return 0;
}

File: C#-Language/FlowControl/ifStatement.cs
using System;

namespace Conditional {
    class IfStatement {
        public static void Main(string[] args) {
            int number = 2;
            if (number < 5) {
                Console.WriteLine("{0} is less than 5", number);
            }

            Console.WriteLine("This statement is always executed.");
        }
    }
}
